src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -259,9 +259,12 @@ void mongoInit
   // "If you call multiple ensureIndex() methods with the same index specification at the same time,
   // only the first operation will succeed, all other operations will have no effect."
   //
+  if (!mtenant)
+  {
   ensureLocationIndex("");
   ensureDateExpirationIndex("");
```

Comment:
```suggestion
    ensureLocationIndex("");
    ensureDateExpirationIndex("");
```

To follow style rule https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/contribution_guidelines.md#m4-indent

Reply:
I have updated the file